site/tutorials/keptn-litmus-018.md

```diff
@@ -0,0 +1,378 @@
+
+To configure a Git upstream for this tutorial, the Git user (`--git-user`), an access token (`--git-token`), and the remote URL (`--git-remote-url`) are required. If a requirement is not met, go to [the Keptn documentation](https://keptn.sh/docs/0.9.0/manage/git_upstream/) where instructions for GitHub, GitLab, and Bitbucket are provided.
+
+Let's define the variables before running the command:
+
+<!-- bash keptn create project litmus --shipyard=./shipyard.yaml -->
+
+```
+GIT_USER=gitusername
+GIT_TOKEN=gittoken
+GIT_REMOTE_URL=remoteurl
+```
+
+Now let's create the project using the `keptn create project` command.
+
+```
+keptn create project litmus --shipyard=./shipyard.yaml --git-user=$GIT_USER --git-token=$GIT_TOKEN --git-remote-url=$GIT_REMOTE_URL
+```
+
+For creating the project, the tutorial relies on a `shipyard.yaml` file as shown below:
+
+```
+apiVersion: "spec.keptn.sh/0.2.0"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-litmus-chaos"
+spec:
+  stages:
+    - name: "chaos"
+      sequences:
+        - name: "delivery"
+          tasks:
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "direct"
+            - name: "test"
+              properties:
+                teststrategy: "performance"
+            - name: "evaluation"
+```
+
+In the `shipyard.yaml` shown above, we define a single stage called *chaos* with a single sequence called *delivery*. In this sequence, a *deployment*, *test*, and *evaluation*  task is defined (along with some properties). With this, Keptn sets up the environment and makes sure, that tests are triggered after each deployment, and the tests are then evaluated by Keptn quality gates. As we do not have a subsequent stage, we do not need an *approval* or *release* task.
+
+## Create a service
+Duration: 2:00
+
+After creating the project, services can be created for our project.
+For this purpose we need the helm charts as a tar.gz archive. To archive it use following command:
+
+<!-- command -->
+```
+cd helloservice && tar cfvz ./helm.tgz ./helm && cd ..
+```
+
+1. Create the **helloservice** service using the [keptn create service](https://keptn.sh/docs/0.10.x/reference/cli/commands/keptn_create_service/) and [keptn add-resource](https://keptn.sh/docs/0.10.x/reference/cli/commands/keptn_add-resource/)commands:
+
+    <!-- command -->
+    ```
+    keptn create service helloservice --project=litmus
+    keptn add-resource --project=litmus --service=helloservice --all-stages --resource=./helloservice/helm.tgz --resourceUri=helm/helloservice.tgz
+    ```
+
+1. After creating the service, tests need to be added as basis for quality gates. We are using JMeter tests, as the JMeter service comes "batteries included" with our Keptn installation. Although this could be changed to other testing tools, we are going with JMeter in this tutorial. Let's add some JMeter tests as well as a configuration file to Keptn.
```

Comment:
I think JMeter is included with Keptn 0.18.x but is not included with 0.19.x and later releases.  I'm not sure how to handle this.  You could just say "JMeter is included with in our Keptn 0.18.x installation".  And you could add "but must be installed in later releases".  On the one hand, you don't want to bog your tutorial down with a lot of excess noise but, on the other hand, people have a bad habit of using a tutorial for different releases.